From patches Mon Sep 17 00:00:00 2001
From: mrhines@linux.vnet.ibm.com
To: qemu-devel@nongnu.org
Cc: Anthony Liguori <aliguori@us.ibm.com>, mst@redhat.com,
 Michael R Hines <mrhines@us.ibm.com>, Bulent Abali <abali@us.ibm.com>,
 owasserm@redhat.com, pbonzini@redhat.com, Gokul B Kandiraju <gokul@us.ibm.com>
Date: Wed, 10 Apr 2013 18:28:16 -0400
Message-id: 1365632901-15470-9-git-send-email-mrhines@linux.vnet.ibm.com
Subject: [PATCH 08/13] default chunk registration to true

From: "Michael R. Hines" <mrhines@us.ibm.com>

Just enable it by default. User can now disable if they want to.

Signed-off-by: Michael R. Hines <mrhines@us.ibm.com>
---
 migration.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/migration.c b/migration.c
index 404c19a..41cf5ba 100644
--- a/migration.c
+++ b/migration.c
@@ -69,6 +69,7 @@ MigrationState *migrate_get_current(void)
         .state = MIG_STATE_SETUP,
         .bandwidth_limit = MAX_THROTTLE,
         .xbzrle_cache_size = DEFAULT_MIGRATE_CACHE_SIZE,
+        .enabled_capabilities[MIGRATION_CAPABILITY_CHUNK_REGISTER_DESTINATION] = true,
     };
 
     return &current_migration;
-- 
1.7.10.4

